lib/show_n_tell/repo.ex

```diff
@@ -5,6 +5,11 @@ defmodule ShowNTell.Repo do
   Dynamically loads the repository url from the
   DATABASE_URL environment variable.
   """
+
+  defmodule Repo do
```

Comment:
We already define `ShowNTell.Repo` up top, so we probably don't need another repo with `my_app` in it. This line already exists: `use Ecto.Repo, otp_app: :show_n_tell`